proto_v2/slot_v2.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+package fd.v2;
+
+import "txn_v2.proto";
+
+/* High level of what happens in a slot:
+   1. Update sysvars
+   2. Distribute partitioned rewards to stake accounts
+   3. Collect raw txns from the ledger/network. TODO: Figure out if we should
+      fuzz using the raw txn bytes or the partially parsed TxnEnvs. Perhaps
+      leave this as a configuration option. For the sake of simplicity as a 
+      PoC, will assume that we pass in the partially parsed TxnEnvs.
+   4. TODO: Decide if we want to fuzz the ticks/poh??? Assuming this is not being
+      done for the sake of simplicity as we are just trying to fuzz the runtime.enum
+      However, this is possible to do.
+   5. Pass in all txns into waves that are generated, or use the pack code path
+   6. Execute each transaction in the wave.
+   7. Finalize block: update sysvar(s) update bpf cache, update
+      bank hash. Collect Rent too. Collect slot fees.
+   8. As a note: this should also be able to handle fuzzing of the epoch
+      boundary.
+*/
+
+message SlotEnv {
+   /* Slot number for the given slot. */
+   uint64            slot_number           = 1;
+
+   /* Previously executed slot. */
+   uint64            prev_slot             = 2;
+
+   /* Block height for the given slot. */
+   uint64            block_height          = 3;
+
+   /* Previous lamports per signature. */
+   uint64            prev_lamports_per_sig = 4;
+
+   /* Number of signatures in the previous slot. */
+   uint64            parent_signature_cnt  = 5;
+
+   /* Fee rate for the slot that is determined at the epoch boundary. */
+   uint64            fee_rate              = 6;
+
+   /* Last restart slot. */
+   uint64            last_restart_slot     = 7;
+
+    /* Vector of account pubkeys that is set at the epoch boundary. */
+   bytes             vote_accounts         = 8; 
+   bytes             stake_accounts        = 9;
```

Comment:
maybe these could be indices within the account states instead

Reply:
set of accounts may change across slots and keeping it as pubkey will make it easier to access via funk